### DISCUSSION TITLE
Opentelemetry-javaagent & JMX Insights

### DISCUSSION THREAD

**Discussion by rbcksantoro** [open-telemetry/opentelemetry-java-instrumentation#8342] [Q&A]
Can i use JMX insights with the standard opentelemetry-javaagent? If so what configuration parameters do i need to add in order to enable this?

**Comment by trask** [comment]
hi @rbcksantoro! check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent#jmx-metric-insight
